src/components/Header/Header.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon } from 'components/Icon/Icon';
+import Logo from 'components/Logo/Logo';
+
+const HeaderDiv = styled.header`
+  display: flex;
+  width: 100vw;
+  height: 60px;
+  background-color: ${props => props.theme.background.light};
+  padding: 15px 20px 15px 20px;
+  align-items: center;
+  justify-content: space-between;
+
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    height: 80px;
+  }
+`;
+
+const LogoutDiv = styled.div`
+  display: flex;
+  gap: 4px;
+  height: 30px;
+  align-items: center;
+  fill: ${props => props.theme.colors.logoutButton};
+  font-size: 18px;
+  font-weight: 400;
+
+  svg {
+    width: 18px;
+    height: 18px;
+  }
+
+  .exitText {
+    @media (max-width: ${props => props.theme.breakpoints.tablet}) {
+      display: none;
+    }
+  }
+
+  .nameText {
+    margin-right: -4px;
+    @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+      margin-right: 0px;
+    }
+  }
+
+  .exitButton {
+    display: flex;
+
+    span {
+      margin-left: 6px;
+    }
+  }
+
+  .divider {
+    height: 30px;
+    margin-right: 4px;
+    border: 1px solid ${props => props.theme.colors.logoutButton};
+    @media (max-width: ${props => props.theme.breakpoints.tablet}) {
+      display: none;
+    }
+  }
+
+  button {
+    border: none;
+    background-color: transparent;
+    color: ${props => props.theme.colors.logoutButton};
+  }
+`;
+
+const Header = () => {
+  return (
+    <HeaderDiv>
+      <Logo></Logo>
+      <LogoutDiv>
+        <span className="nameText">
+          <button>Name</button>
+        </span>
+        <div className="divider"></div>
+
+        <button className="exitButton">
```

Comment:
brak type="button" w buttonach, wiem że pierdoła ale skoro wszystko idzie przez pull requesty to proszę o poprawkę bo nie chce wysyłać commita prosto w dev
dorzucam info odnośnie type
![image](https://github.com/pawelrogowski/goit-wallet-app-03-frontend/assets/110867327/5f449ae0-f5a2-495b-9e88-402a39470a08)